packages/slate/test/models/by-key/get-furthest-only-child.js

```diff
@@ -0,0 +1,20 @@
+/** @jsx h */
+
+import h from '../../helpers/h'
+
+const { document } = (
+  <value>
+    <document>
+      <paragraph>Some Text</paragraph>
+    </document>
+  </value>
+)
+const pNode = document.nodes.first()
+const textNode = pNode.getFirstText()
+
+export const actual = [
+  document.getFurthestOnlyChildAncestor(pNode.key),
+  pNode.getFurthestOnlyChildAncestor(textNode.key),
+  document.getFurthestOnlyChildAncestor(textNode.key),
+]
+export const expected = [null, null, pNode]
```

Comment:
Instead of structuring this as `actual` and `expected`, can you change it to import the `assert` module directly and expose a single `test` function instead that does the assertions directly?

Reply:
@ianstormtaylor  Fixed~ Thank you.